source/LoadPanel.cpp

```diff
@@ -441,6 +441,7 @@ void LoadPanel::UpdateLists()
 	files.clear();
 
 	vector<string> fileList = Files::List(Files::Saves());
+	set<string> notSnapshotOnly;
```

Comment:
So these are saves that are not snapshots?
The name is a bit confusing to me.

Reply:
Using @quyykk's suggestion and removing the `set` anyway.